packages/common/src/index.js

```diff
@@ -55,14 +55,15 @@ const TOPICS = [
   'Equity',
   'Facilities',
   'Family Support Services',
+  'Fatherhood / Male Caregiving',
   'Fiscal / Budget',
   'Five-Year Grant',
   'Home Visiting',
   'Human Resources',
   'Leadership / Governance',
   'Learning Environments',
   'Nutrition',
-  'Ongoing Monitoring Management System',
```

Comment:
Does it still make sense to use this list instead of fetching topics from the backend?

Reply:
We can create a new ticket to cleanup all the various spots. This will at least get them back to a working state.

Reply:
Reminder to create that ticket

Reply:
[TTAHUB-2236](https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2236)